src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1791,49 +1758,37 @@
     short-summary: Create a peering.
     examples:
         - name: Create a virtual network peering between virtual networks in the same region
-          text: >
+          text: |
             az network vnet create --name myVnet1 --resource-group myResourceGroup --location eastus --address-prefix 10.0.0.0/16
-            \\n\\n az network vnet create --name myVnet2 --resource-group myResourceGroup --location eastus --address-prefix 10.1.0.0/16
-            \\n\\n vnet1Id=$(az network vnet show --resource-group myResourceGroup--name myVnet1 --query id --out tsv)
-            \\n\\n vnet2Id=$(az network vnet show --resource-group myResourceGroup --name myVnet2 --query id --out tsv)
-            \\n\\n az network vnet peering create --name myVnet1ToMyVnet2 --resource-group myResourceGroup --vnet-name myVnet1 --remote-vnet-id $vnet2Id --allow-vnet-access
-            \\n\\n az network vnet peering create --name myVnet2ToMyVnet1 --resource-group myResourceGroup --vnet-name myVnet2 --remote-vnet-id $vnet1Id --allow-vnet-access
+            az network vnet create --name myVnet2 --resource-group myResourceGroup --location eastus --address-prefix 10.1.0.0/16
```

Comment:
What does this look like with -h?  We need to be able to differentiate multi-step examples (which are relatively rare and arguably not what this section is intended for) from multiple examples.  When steps of a multi-step example wrap, there should be some kind of hanging indent.  If this isn't supported, please open an issue against help so we can work on that.

Reply:
Is there a quick way to build and run an in-development CLI branch to check the `-h` output like this?

Reply:
https://prompt.ws/r/Azure/azure-cli/4834
 A bot creates one of these for every internal PR. Compliments of the great @derekbekoe

Reply:
Keep getting a connection error. CI failure, or a permissions problem, @derekbekoe ?